recipes/itemadapter/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "itemadapter" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 52159b4f97d82aa2968000ee8371b2114af56a2f44e4cd9142580d46eea39020
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - itemadapter
```

Can you also try to add ``pip check`` please?
```suggestion
    - itemadapter
  requires:
    - pip
  commands:
    - pip check
```